week7/cooper_client_react/12_add_mocks.md

```diff
@@ -1,86 +1,82 @@
+# Add mocks for our acceptance tests
+
+## How to mock with cypress
```

Comment:
Cypress is a product name, please use a capital letter.

Reply:
addressed